src/test/java/racingcar/ApplicationTest.java

```diff
@@ -31,6 +31,49 @@ class ApplicationTest extends NsTest {
         );
     }
 
+    @Test
+    void 입력받은_이름_공백_제거() {
+        assertRandomNumberInRangeTest(
+                () -> {
+                    run("pobi,  woni  ", "1");
+                    assertThat(output()).contains("pobi : -", "woni : ", "최종 우승자 : pobi");
+                },
+                MOVING_FORWARD, STOP
+        );
+    }
+
+    @Test
+    void 유효한_이름이_없다면_예외_처리() {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException(" , ", "1"))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+    }
+
+    @Test
+    void 이동_횟수_범위밖_예외_처리() {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException("a", "11"))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException("a", "0"))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+    }
+
+    @Test
+    void 여러명_우승자_출력() {
+        assertRandomNumberInRangeTest(
+                () -> {
+                    run("pobi,woni", "1");
+                    assertThat(output()).contains("pobi : -", "woni : ", "최종 우승자 : pobi, woni");
```

Comment:
아래 처럼 `"woni : -"` 로 검증하지 않으신 이유가 있으실까요~?
여러명 우승자 출력이라면, pobi 와 같은 값이어야 하지 않나 싶은 의문이 들었어요.
실제 출력 결과가 "woni : -" 이라, 앞 "woni : " 부분은 포함되니 통과는 되지만요..!

```suggestion
                    assertThat(output()).contains("pobi : -", "woni : -", "최종 우승자 : pobi, woni");
```

Reply:
아.. 이 부분은 제가 실수했네요 ㅠㅠ